sdk/core/System.BinaryData/tests/System.BinaryData.Tests.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  
+  <PropertyGroup>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+    <IsTestProject>true</IsTestProject>
+  </PropertyGroup>
+  
+  <ItemGroup>
+    <PackageReference Include="Moq" />
```

Comment:
Please omit the usage of Moq (already suggested/requested with a using, but seems more appropriate here).